src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairHMM.java

```diff
@@ -101,12 +101,16 @@ public VectorLoglessPairHMM() throws UserException.HardwareFeatureException {
                     logger.info("libVectorLoglessPairHMM found in JVM library path");
                 } catch (UnsatisfiedLinkError e) {
                     //Could not load from Java's library path - try unpacking from jar
-                    logger.debug("libVectorLoglessPairHMM not found in JVM library path - trying to unpack from GATK jar file");
-                    String path = "/lib/libVectorLoglessPairHMM" + (runningOnMac ? ".dylib" : ".so");
-                    File temp = IOUtils.writeTempResource(new Resource(path, VectorLoglessPairHMM.class));
-                    temp.deleteOnExit();
-                    System.load(temp.getAbsolutePath());
-                    logger.info("libVectorLoglessPairHMM unpacked successfully from GATK jar file");
+                    try {
+                        logger.debug("libVectorLoglessPairHMM not found in JVM library path - trying to unpack from GATK jar file");
+                        String path = "/lib/libVectorLoglessPairHMM" + (runningOnMac ? ".dylib" : ".so");
+                        File temp = IOUtils.writeTempResource(new Resource(path, VectorLoglessPairHMM.class));
+                        temp.deleteOnExit();
+                        System.load(temp.getAbsolutePath());
+                        logger.info("libVectorLoglessPairHMM unpacked successfully from GATK jar file");
+                    } catch (final Exception furtherException){
+                        throw new UserException.HardwareFeatureException("Could not load AVX PairHMM library from GATK jar", furtherException);
```

Comment:
Adjust this error message slightly to make it clearer what the situation is -- eg., "Machine supports AVX, but failed to load AVX PairHMM library from classpath"